docs/quickstart.md

```diff
@@ -85,6 +85,8 @@ The chart works with both Helm 2 and Helm 3. The `crd-install` hook from v2 will
 be skipped with warning when using v3. Documentation for installing applications
 with Helm 2 can be found in the [v2 docs](https://v2.helm.sh/docs/).
 
+The chart can also be found at: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
```

Comment:
This URL currently gives a 404. Is this something which is supposed to work after this is merged?

Reply:
```suggestion
The chart can also be found at: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/postgres-operator-1.6.3.tgz
```

Reply:
this is the url of the chart repo, helm uses it to query the index.yaml: 
```
https://opensource.zalando.com/postgres-operator/charts/postgres-operator/index.yaml
```
and from there, retrieve the path of the chart and append it to the same prefix to download.

Reply:
@FxKu please revert your change

Reply:
the way you install chart is for example:
```
helm repo add postgres-operator https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
helm install postgres-operator/postgres-operator postgres-operator ...
```
If you do not want the 404 you can make it redirect to the github repo are something like that, but it is really not a problem

Reply:
Then I suggest these are the commands you shall add to the quickstart. Otherwise it's confusing.

Reply:
@FxKu this standard when using helm it is also not the only way, if you use helmfile or other your file looks like:
repositories:
  - name: postgres-operator
    url: https://opensource.zalando.com/postgres-operator/charts/postgres-operator/
releases:
  - chart: postgres-operator/postgres-operator
    namespace: postgres-operator
    version: 1.6.3
    name: postgres-operator

Reply:
@FxKu that is good with me :) thanks
Somehow CI keeps failing though...